docs/animation.md

```diff
@@ -0,0 +1,19 @@
+> Transitions & Animation
+
+# Animation
+Utilities for animating elements with CSS animations.
+
+## Quick reference
+
+<qr-table />
+
+## Basic usage
+### Spin
+Add the animate-inprogress utility to add a animated background to indigate that something is in progress.
```

Comment:
Is the class `animate-inprogress` or `in-progress`?

Reply:
> Is the class `animate-inprogress` or `in-progress`?

animate-

My thinking was to copy TW's setup for small animations like this.... 
https://tailwindcss.com/docs/animation 

This would give all those odd little thingies a place to live :)